rosetta-source/src/main/rosetta/product-qualification-func.rosetta

```diff
@@ -629,9 +629,7 @@ func Qualify_EquityOption_PriceReturnBasicPerformance_SingleName: <"Qualifies a
                 // qualifies that the underlier is an equity security
             and economicTerms -> payout -> optionPayout only-element -> underlier -> security exists
                 // qualifies that an option style is defined and that no feature other than averaging exists
-            and (economicTerms -> payout -> optionPayout -> exerciseTerms -> optionStyle -> americanExercise exists
-                or economicTerms -> payout -> optionPayout -> exerciseTerms -> optionStyle -> europeanExercise exists
-                or economicTerms -> payout -> optionPayout -> exerciseTerms -> optionStyle -> bermudaExercise exists)
+            and (economicTerms -> payout -> optionPayout -> exerciseTerms -> style exists)
```

Comment:
This condition is redundant because `exerciseTerms` and `style` are both mandatory on `OptionPayout`.